src/components/dashboard.component.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { withRouter } from "react-router";
+import axios from "axios";
+import * as Constants from "../constants";
```

Comment:
It may be worth importing `GET_USER` explicitly if we only need one value from the constants file.